index2.html

```diff
@@ -0,0 +1,230 @@
+                  <li class="header-list" id="frameworks">
+                    <img id="titles" src="img/Rectangle 56.png" alt="Languages">
+                    <h1>Frameworks</h1>
+                  </li>
+                  <ul class="mini-list">
+                    <li class="detail">React.js</li>
+                    <li class="detail">Ruby on rails</li>
+                    <li class="detail">RSpec</li>
+                    <li class="detail">Capybara</li>
+                  </ul>
+                 </ul>
+                 <ul class="listtopography">
+                    <li class="header-list" id="skills">
+                      <img id="titles" src="img/Rectangle 57.png" alt="Languages">
+                      <h1>Skills</h1>
+                     </li>
+                     <ul class="mini-list">
+                      <li class="detail">Database Management</li>
+                      <li class="detail">Version Control</li>
+                      <li class="detail">CLI</li>
+                      <li class="detail">API Design</li>
+                      <li class="detail">Web Development</li>
+                     </ul>
+                  </ul>
+              </ul>
+          </div>
+       </section>
+       <section class="bottom-footer">
+           <div class="contact-section">
+              <p class="contact-text"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+           </div>
+           <form action="https://formspree.io/html" method="post">
+            <fieldset> 
+            <label> 
+           <input id="Full-name" name="" type="text" required placeholder="Full name" max-length="30" />
+            </label>
+            <label>
+           <input id="email" name="" type="email" required placeholder="Email address" />
+          </label>
+          </fieldset>
+            <textarea name="" id="mess" placeholder="Write me something..." cols="50" rows="10" maxlength="500"></textarea>
+         <button class="Submit-btn">
+           Get in touch
+         </button>
+        </form>
+        <hr class="form-breaker">
+        <div class="footer">
+            <ul class="footerIcons">
+                <li class="icons">
+                    <a href=""><img class="icon-bg" src="img/github.svg" alt="GitHub" /></a>
+                </li>
+                <li class="icons">
+                    <a href=""><img class="icon-bg" src="img/Linkedin icon.svg" alt="GitHub" /></a>
+                </li>
+                <li class="icons">
+                    <a href=""><img class="icon-bg" src="img/angel list.svg" alt="GitHub" /></a>
+                </li>
+                <li class="icons">
+                    <a href=""><img class="icon-bg" src="img/twitter.svg" alt="GitHub" /></a>
+                </li>
+                <li class="icons">
+                    <a href=""><img class="icon-bg" src="img/medium.svg" alt="GitHub" /></a>
+                </li>
+            </ul>
```

Comment:
- [ ] Lastly, just as the previous reviewer mentioned, please ensure that your `social icons` have the same colors as seen in the Figma design :wink: 
- Your Design
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/102699958/1f3950c8-7279-4d67-a11a-dbe05d2a723b)

- The Figma Design
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/102699958/102e5796-a976-4e49-bdda-dedabe62b66c)